.travis.yml

```diff
@@ -0,0 +1,17 @@
+language : node_js
+
+node_js: node_js
+
+cache: npm
+
+install:
+  -npm ci
+
+script:
+  -npm test
+
+deploy:
+  provider: pages
+  github-token: $GITHUB_TOKEN
+  on:
+    branch: master  
```

Comment:
Add a newline here or better still add this line to your VSCode `settings.json` file `  "files.insertFinalNewline": true,`